assets/sass/_top-button.scss

```diff
@@ -0,0 +1,26 @@
+.top-button {
```

Comment:
I am not a fan of the current style of the button per the screenshot, but I don't have a recommendation either. I am sure Christian would have some ideas, though.

Reply:
neither am I, tbh.